src/lib/learn-client/api/collection/index.ts

```diff
@@ -141,7 +143,8 @@ export async function getCollectionsBySection(section: string) {
 			console.warn(`No collections found for section query: ${section}`)
 		}
 
-		return res.result
+		/** Format and return collections */
+		return res.result.map(formatCollection)
```

Comment:
Previous lack of formatting meant, for example, `collection.shortName` for sidebar use wasn't available (was not reformatted, so was only available under `collection.short_name`).